app/views/works/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_with model: @work, class: 'work-form' do |f| %>
+
+<h3>Create New Media Form</h3>
+
+<div class="work-form">
+  <%= f.label :title %>
+  <%= f.text_field :title %>
+  <%= f.label :creator %>
+  <%= f.text_field :creator %>
+  <%= f.label :publication_year %>
+  <%= f.text_field :publication_year %>
+  <%= f.label :description %>
+  <%= f.text_field :description %>
+  <%= f.label :category %>
+  <%= f.select :category, Work::CATEGORIES %>
+  <%= f.submit action_name, class: "work btn" %>
+  <% end %>
+</div>
```

Comment:
You end up having essentially a form duplicate to the edit form... Why didn't you use the partial? Would that have also worked?